src/org/usfirst/frc/team467/robot/Autonomous/Actions.java

```diff
@@ -671,7 +641,6 @@ public static ActionGroup rightAdvancedSwitchLeftScaleLeft() {
 		mode.addActions(turn(90));
 		mode.addAction(grabCube());
 		mode.addActions(move(1.3));
-
```

Comment:
Probably should leave the blank line here, and maybe add a comment because it's really confusing why we drive forward then back again -- it looks like a bug unless you understand the why.